### PR TITLE
🐛 Fix referencing to deleted "provision_host.sh" script

### PR DESCRIPTION
**Pull request by furkatgofurov7**
The `provision_host.sh` script was part of v1alpha1 scripts and was removed by [this patch ](https://github.com/metal3-io/metal3-dev-env/pull/322)

### CODE REVIEW

File: vars.md [hunk 1]
```diff
@@ -11,19 +11,19 @@ assured that they are persisted.
...
 | EXTERNAL_SUBNET                | This is the subnet used on the "baremetal" libvirt network, created as the primary network interface for the virtual bare metalhosts.                                                                                                                    | CIDR                               | 192.168.111.0/24                                             |
-| SSH_PUB_KEY                    | This SSH key will be automatically injected into the provisioned host by the provision_host.sh script.                                                                                                                                                   |                           | ~/.ssh/id_rsa.pub                                            |
+| SSH_PUB_KEY                    | This SSH key will be automatically injected into the provisioned host by the lib/common.sh script.                                                                                                                                                   |                           | ~/.ssh/id_rsa.pub                                            |
```
Comment (namnx228) [inline vars.md:14]: Could you please refer to code location where this happens?

**Review by namnx228 — COMMENTED** [review]
Look good to me. Only one thing left is the thing I commented

File: vars.md (hunk 1 above)
Comment (furkatgofurov7) [reply vars.md:14]: Sure, [here](https://github.com/metal3-io/metal3-dev-env/blob/master/lib/common.sh#L60) it is

Comment (namnx228) [reply vars.md:14]: I think we declare the key in `lib/common.sh`, but this key will be automatically injected into the provisioned host by another script, no?

Comment (furkatgofurov7) [reply vars.md:14]: Which script do you mean by another script?

Comment (furkatgofurov7) [reply vars.md:14]: Current `01_prepare_host.sh` is what `provision_host.sh` had been in my opinion, is that what you meant?

Comment (namnx228) [reply vars.md:14]: No, I mean the script which does something like `ssh-add <key><host>`. If I am not mistaken, script 02 or 03 do that.

Comment (furkatgofurov7) [reply vars.md]: You were right, I have changed it, thanks for review!

**Review by namnx228 — APPROVED** [review]

**Review by maelk — APPROVED** [review]

**Pull request merged**